php.ini-production

```diff
@@ -1457,7 +1443,7 @@ session.hash_function = 0
 ; Development Value: 5
 ; Production Value: 5
 ; http://php.net/session.hash-bits-per-character
```

Comment:
The link needs to get updated, but the shortcut http://php.net/session.sid-bits-per-character doesn't seem to exist yet.